tests/e2e/test_e2e.py

```diff
@@ -0,0 +1,97 @@
+import subprocess
+import sys
+from pathlib import Path
+
+import pandas as pd
+import pytest
+import yaml
+
+
+@pytest.fixture
+def model_spec(tmp_path: Path) -> Path:
+    model_spec_path = (
+        Path(__file__).parent.parent.parent
+        / "src"
+        / "vivarium_gates_nutrition_optimization"
+        / "model_specifications"
+        / "model_spec.yaml"
+    )
+    with open(model_spec_path, "r") as file:
+        ms = yaml.safe_load(file)
+
+    # modify the time step size so there are only 3 or 4 steps
```

Comment:
Is this what we want to do? Sometimes dramatically increasing the step-size can introduce or hide bugs.

Reply:
What kind of bugs? Seems like we just interpolate between end points so shouldn't things still work?

Regardless, conversely, won't we constantly run into small numbers issues if we keep the time step very small but only run through a few timesteps? I was trying to keep this very general so that it can one day be the standard, but maybe that's just not possible and we need to run a custom test sim for every model repo?